### PR TITLE
Biopdu-v1.1.0

### PR DESCRIPTION
**Pull request by ZioFabry**
## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ ] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

File: tasmota/tasmota_xx2c_global/xnrg_interface.ino [hunk 1]
```diff
@@ -152,6 +152,10 @@ bool (* const xnrg_func_ptr[])(uint32_t) = {   // Energy driver Function Pointer
 #ifdef XNRG_32  // Reserved for use by xdrv_16_tuyamcu.ino
   &Xnrg32
 #endif
+
```
Comment (arendst) [inline tasmota/tasmota_xx2c_global/xnrg_interface.ino:155]: Don't change. Use xnrg_24 instead.

**Comment by ZioFabry** [comment]
Biomine BioPDU 625x12 Driver

This custom board is used to build an Industrial PDU specific for the High-End Crypto Mining Market:

Base board BioPDU 625x12
1 Olimex ESP32-POE
1 x MCP23008 on board
6 x PZEM004T
3Bit Serial Mux on board
6 x 25A relay
6 x 25A circuit breaker
8-Port Ethernet Switch Integrated

This driver use the new 8-phase energy driver.

File: tasmota/tasmota_xx2c_global/xnrg_interface.ino (hunk 1 above)
Comment (ZioFabry) [reply tasmota/tasmota_xx2c_global/xnrg_interface.ino:155]: thanks.... done.

Comment (Jason2866) [reply tasmota/tasmota_xx2c_global/xnrg_interface.ino:155]: Really?

Comment (ZioFabry) [reply tasmota/tasmota_xx2c_global/xnrg_interface.ino:155]: yes... do u want see it ? :)

Comment (ZioFabry) [reply tasmota/tasmota_xx2c_global/xnrg_interface.ino]: it's ok now ?

**Comment by ZioFabry** [comment]
sorry, this is my first PR, I missed to updated somethings ?

**Comment by Jason2866** [comment]
Remove the Draft status. As long it is set no merge will be done.

**Comment by ZioFabry** [comment]
Some pictures...

photo_2023-02-02_18-59-39
BioPDU_Tasmota
photo_2023-02-02_18-57-19
photo_2023-02-02_18-57-32

**Pull request merged**